### DISCUSSION TITLE
New addons (Toggle native tab bar & TST Active and Total Tabs Counter)

### DISCUSSION THREAD

**Discussion by irvinm** [piroor/treestyletab#3581] [General]
# TST Active and Total Tabs Counter
I have recently added a new addon that is intended to solve the recent TST changes that made the CSS tab counting solutions ineffective.  This addon tracks the active tabs (non-discarded) and the total tabs but also shows them *BY* window if you are a multi-window user of Firefox.

- AMO: https://addons.mozilla.org/en-US/firefox/addon/tst-active-and-total-tabs-ctr/
- Github: https://github.com/irvinm/TST-Active-and-Total-Tabs-Counter

![301183](https://github.com/piroor/treestyletab/assets/979729/b59104df-8167-400f-b9f0-f56afd787007)

# Toggle Native Tab Bar
As the stated desire is for the native tab bar to be available at all times as the TST sidebar can not emulate everything, I used an idea from another addon and have come up with a way to simply toggle the native tab bar so that it can been hidden most of the time and shown when needed.  The addon uses a little userChrome.css to facilitate this behavior.

- AMO: https://addons.mozilla.org/en-US/firefox/addon/toggle-native-tab-bar/
- Github (including instructions): https://github.com/irvinm/Toggle-Native-Tab-Bar

![Demo](https://github.com/piroor/treestyletab/assets/979729/a60e842c-554d-46bd-b3c4-e45820002c51)

@piroor I added the counter addon to the 3rd party addons that extend TST, but wasn't sure if you would want the native tab bar one there too.

**Comment by piroor** [comment]
No problem adding it to the list! I've done it and synchronized the updated list to the ja localized version. Thanks!
